templates/OperationRequestBodySnippet.stencil

```diff
@@ -22,8 +22,8 @@ guard let requestBody = try? JSONEncoder().encode(body) else {
     return
 }
 {% endif %}
-guard let requestUrl = url.appendingQueryParameters(queryParams) else {
-    self.options.logger.error("Failed to append query parameters to URL")
+guard let requestUrl = self.url(forTemplate: urlTemplate, withKwargs: pathParams, and: queryParams) else {
```

Comment:
The `url` method in AzureCore does not have this `and` parameter. What is it for?

Reply:
I create a helper method so that it builts an URL based on urltemplate and query parameters. This is different from AzureCore , which has 1 method to build url and another to add query parameters.